packages/term/ansi_term.pony

```diff
@@ -154,6 +186,20 @@ actor ANSITerm
     """
     _notify.prompt(this, value)
 
+  be size() =>
+    _size()
+
+  fun ref _size() =>
+    """
+    Pass the window size to the notifier.
+    """
+
```

Comment:
I don't think it's very typical in this codebase to use an empty line between method docstring and first line of code.